pkg/segment/utils/segconsts.go

```diff
@@ -300,6 +301,7 @@ const (
 	Estdc
 	EstdcError
 	ExactPerc
+	Mean
```

Comment:
We already have `Avg`; why do we need `Mean`?

Reply:
addressed in the latest commit.